project/AkkaBuild.scala

```diff
@@ -400,7 +137,20 @@ object AkkaBuild extends Build {
     testOptions in Test += Tests.Argument("-oDF"),
 
     // don't save test output to a file
-    testListeners in (Test, test) := Seq(TestLogger(streams.value.log, {_ => streams.value.log }, logBuffered.value)),
```

Comment:
The TestLogger in sbt is deprecated since a while back, no documented replacement, so I tried to just duplicate the logic of it, but I guess I misunderstood something it does.